{"uid":"3739b7cf209319e4","name":"test_backup_ceph","fullName":"tests.integration.test_backups_ceph#test_backup_ceph","historyId":"532c16ceec0f52085db049778cec102e","time":{"start":1761617522160,"stop":1761618533726,"duration":1011566},"description":"Build and deploy two units of PostgreSQL in microceph, test backup and restore actions.","descriptionHtml":"<p>Build and deploy two units of PostgreSQL in microceph, test backup and restore actions.</p>\n","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\n  postgresql-ceph/0 [allocating] waiting: waiting for machine\n  postgresql-ceph/1 [allocating] waiting: waiting for machine","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1671, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 157, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_backups_ceph.py\", line 175, in test_backup_ceph\n    await backup_operations(\n  File \"/root/spread_project/tests/integration/helpers.py\", line 1149, in backup_operations\n    await ops_test.model.wait_for_idle(apps=[database_app_name], status=\"active\", timeout=1000)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3250, in wait_for_idle\n    raise asyncio.TimeoutError(\nTimeoutError: Timed out waiting for model:\n  postgresql-ceph/0 [allocating] waiting: waiting for machine\n  postgresql-ceph/1 [allocating] waiting: waiting for machine","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1761617477378,"stop":1761617477379,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"microceph","time":{"start":1761617477379,"stop":1761617521794,"duration":44415},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"cloud_configs","time":{"start":1761617521794,"stop":1761617521806,"duration":12},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"cloud_credentials","time":{"start":1761617521806,"stop":1761617521806,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1761617521807,"stop":1761617521807,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1761617521807,"stop":1761617521807,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1761617521807,"stop":1761617522159,"duration":352},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1761617522159,"stop":1761617522159,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Build and deploy two units of PostgreSQL in microceph, test backup and restore actions.","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\n  postgresql-ceph/0 [allocating] waiting: waiting for machine\n  postgresql-ceph/1 [allocating] waiting: waiting for machine","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1671, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 157, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_backups_ceph.py\", line 175, in test_backup_ceph\n    await backup_operations(\n  File \"/root/spread_project/tests/integration/helpers.py\", line 1149, in backup_operations\n    await ops_test.model.wait_for_idle(apps=[database_app_name], status=\"active\", timeout=1000)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3250, in wait_for_idle\n    raise asyncio.TimeoutError(\nTimeoutError: Timed out waiting for model:\n  postgresql-ceph/0 [allocating] waiting: waiting for machine\n  postgresql-ceph/1 [allocating] waiting: waiting for machine","steps":[],"attachments":[{"uid":"4ee758c3c55f082d","name":"log","source":"4ee758c3c55f082d.txt","type":"text/plain","size":9664}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1761618533738,"stop":1761618533738,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1761618533739,"stop":1761618534070,"duration":331},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1761618534071,"stop":1761618534071,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1761618534071,"stop":1761618534071,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1761618534071,"stop":1761618534071,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_backups_ceph"},{"name":"host","value":"runnervmwhb2z"},{"name":"thread","value":"14835-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_backups_ceph"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"41e6bebb9b974353","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":10,"skipped":0,"passed":336,"unknown":21,"total":367},"items":[{"uid":"5bfc43145e2b9fc0","reportUrl":"../5904//#testresult/5bfc43145e2b9fc0","status":"passed","time":{"start":1761528764394,"stop":1761529483362,"duration":718968}},{"uid":"100d513a56a77e56","reportUrl":"../5901//#testresult/100d513a56a77e56","status":"passed","time":{"start":1761442382591,"stop":1761443083413,"duration":700822}},{"uid":"511009c95c81e9a","reportUrl":"../5899//#testresult/511009c95c81e9a","status":"passed","time":{"start":1761355988824,"stop":1761356741756,"duration":752932}},{"uid":"a406930ee4e95982","reportUrl":"../5889//#testresult/a406930ee4e95982","status":"passed","time":{"start":1761269590634,"stop":1761270363810,"duration":773176}},{"uid":"a07b9ee32fab3d3b","reportUrl":"../5884//#testresult/a07b9ee32fab3d3b","status":"passed","time":{"start":1761183225828,"stop":1761184002597,"duration":776769}},{"uid":"e2110d92edf1ecea","reportUrl":"../5867//#testresult/e2110d92edf1ecea","status":"passed","time":{"start":1761010374796,"stop":1761011111154,"duration":736358}},{"uid":"8bdb44c5ebe29dd5","reportUrl":"../5850//#testresult/8bdb44c5ebe29dd5","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-ceph/0 [allocating] waiting: waiting for machine\n  postgresql-ceph/1 [allocating] waiting: waiting for machine","time":{"start":1760923944531,"stop":1760924956028,"duration":1011497}},{"uid":"128088c373c2e16","reportUrl":"../5847//#testresult/128088c373c2e16","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-ceph/0 [allocating] waiting: waiting for machine\n  postgresql-ceph/1 [allocating] waiting: waiting for machine","time":{"start":1760837566028,"stop":1760838577346,"duration":1011318}},{"uid":"d6a53afa8d331899","reportUrl":"../5846//#testresult/d6a53afa8d331899","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-ceph/0 [allocating] waiting: waiting for machine\n  postgresql-ceph/1 [allocating] waiting: waiting for machine","time":{"start":1760751129002,"stop":1760752141355,"duration":1012353}},{"uid":"2317764483c250b2","reportUrl":"../5838//#testresult/2317764483c250b2","status":"passed","time":{"start":1760664742368,"stop":1760665460217,"duration":717849}},{"uid":"c7a2009f74945f3c","reportUrl":"../5835//#testresult/c7a2009f74945f3c","status":"passed","time":{"start":1760578291522,"stop":1760579007971,"duration":716449}},{"uid":"1a1048242e9147f","reportUrl":"../5822//#testresult/1a1048242e9147f","status":"passed","time":{"start":1760491932233,"stop":1760492649063,"duration":716830}},{"uid":"7ab087e31024080f","reportUrl":"../5809//#testresult/7ab087e31024080f","status":"passed","time":{"start":1760405517875,"stop":1760406247392,"duration":729517}},{"uid":"a95eb77862c503ea","reportUrl":"../5802//#testresult/a95eb77862c503ea","status":"passed","time":{"start":1760319148726,"stop":1760319869820,"duration":721094}},{"uid":"96ceeb2704cb5d4d","reportUrl":"../5796//#testresult/96ceeb2704cb5d4d","status":"passed","time":{"start":1760232734434,"stop":1760233409455,"duration":675021}},{"uid":"e6029681336b6f06","reportUrl":"../5794//#testresult/e6029681336b6f06","status":"passed","time":{"start":1760146335706,"stop":1760147074419,"duration":738713}},{"uid":"6e0f7fedf0cf2011","reportUrl":"../5787//#testresult/6e0f7fedf0cf2011","status":"passed","time":{"start":1760059970739,"stop":1760060707837,"duration":737098}},{"uid":"d657a7d1c64fe096","reportUrl":"../5776//#testresult/d657a7d1c64fe096","status":"passed","time":{"start":1759973531356,"stop":1759974219897,"duration":688541}},{"uid":"f09f085aaa50314c","reportUrl":"../5764//#testresult/f09f085aaa50314c","status":"passed","time":{"start":1759887128611,"stop":1759887806675,"duration":678064}},{"uid":"e647411d927936f1","reportUrl":"../5745//#testresult/e647411d927936f1","status":"passed","time":{"start":1759800708317,"stop":1759801393756,"duration":685439}}]},"tags":["asyncio"]},"source":"3739b7cf209319e4.json","parameterValues":[]}